src/test/java/step1/domain/StringCalculatorTest.java

```diff
@@ -0,0 +1,44 @@
+package step1.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import step1.domain.StringCalculator;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringCalculatorTest {
+
+    private final String COMMON_INPUT = "2 + 3 * 4 / 2";
+    @Test
```

Comment:
```suggestion
    private final String COMMON_INPUT = "2 + 3 * 4 / 2";

    @Test
```
사소한 거지만 구분을 위해 개행을 해주시는 것이 좋습니다.

Reply:
반영하겠습니다.